vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -102,9 +102,11 @@ def parse_node(node)
         }
       }
 
-      conditions = node.status.conditions
-      new_result[:container_node_conditions] = conditions.collect do |condition|
-        parse_node_condition(condition)
+      unless node.status.conditions.nil?
+        conditions = node.status.conditions
+        new_result[:container_conditions] = conditions.collect do |condition|
```

Comment:
Drop `unless` and just use:

    new_result[:container_conditions] = conditions.to_a.collect do |condition|